### ISSUE TITLE
Update Android System WebView description to make clear the effect of not having a webview package installed

### ISSUE THREAD

**Issue by Dimittrikov** [0x192/Universal-Android-Debloater#573]
**Your phone**: Samsung A21s

**Packages documentation to update:**
```
com.google.android.webview
...
```
## Documentation change
**List**: `Google`|`Misc`|`Oem` (manufacturer)|`Aosp`|`Pending`|`Carrier` (isp).
**Removal**: `Recommended`, `Advanced`, `Expert` (this can break important features), `Unsafe` (this can bootloop the phone or break extremely important features).

### Samsung Device Quality Agent
**List**: `Google` :arrow_right: `Google`
**Removal**: `Advanced` :arrow_right: `Advanced`

**Current description**
>Android System WebView (https://play.google.com/store/apps/details?id=com.google.android.webview)
>Allows Android apps to display content from the web directly inside the app.
>Based on Chrome.
>
>Bromite is an open-source, privacy-oriented Webview replacement: https://www.bromite.org/system_web_view"

**Proposed description**
>Android System WebView (https://play.google.com/store/apps/details?id=com.google.android.webview)
>Allows Android apps to display content from the web directly inside the app.
>Make sure you have a replacement installed if you delete it as many third-party apps will break. 
>Based on Chrome.
>
>Bromite is an open-source, privacy-oriented Webview replacement: https://www.bromite.org/system_web_view"

**Comment by Dimittrikov** [comment]
Didnt know how to properly phrase it there so ill add in a comment

The "third party apps" mentioned is literally any app that has a website. Discord, Github, Twitter, Tumblr, Reddit, anything (tho i have tested only discord github and tumblr)

Forgot to mention, and should probably be used instead of "break", but the apps actually crash on load. They will try to load for a few seconds, like when you dont have internet, then simply crash

**Comment by AnonymousWP** [comment]
> The "third party apps" mentioned is literally any app **that has a website**. Discord, Github, Twitter, Tumblr, Reddit, anything (tho i have tested only discord github and tumblr)

Any app that has a website?? That doesn't make much sense. Don't you mean any app that contains links? I don't think it works like that, as in: if app contains link, then it uses Android System WebView.

> Forgot to mention, and should probably be used instead of "break", but the apps actually crash on load. They will try to load for a few seconds, like when you dont have internet, then simply crash

Agreed, fixed.

**Comment by Dimittrikov** [comment]
Any app that has a website in the sense that it's the mobile app for said website. And the app would make a request to its own website to show you the data. It's not a link. I would assume it's sort of an API request. And apparently that API is WebView. Not being able to find an API, the app crashes
I just can't find a way to phrase this well, nor am I 100% sure about what was happening. That's why I tried giving examples of apps that were affected by it. By contrary, something like Firefox was completely fine.

Also, something that i discovered only later, if Bromite System WebView is going to be mentioned and linked, it should also probably be added the fact that it requires root for installation. At least that's what it's telling me in F-Droid after adding the Bromite repository

**Comment by AnonymousWP** [comment]
> Any app that has a website in the sense that it's the mobile app for said website. And the app would make a request to its own website to show you the data. It's not a link. I would assume it's sort of an API request. And apparently that API is WebView. Not being able to find an API, the app crashes I just can't find a way to phrase this well, nor am I 100% sure about what was happening. That's why I tried giving examples of apps that were affected by it. By contrary, something like Firefox was completely fine.
> 
> Also, something that i discovered only later, if Bromite System WebView is going to be mentioned and linked, it should also probably be added the fact that it requires root for installation. At least that's what it's telling me in F-Droid after adding the Bromite repository

Yeah I also don't really know how to phrase it. I've just done another commit. Could you check?

**Comment by Dimittrikov** [comment]
Yeah I think the description looks much better now. It works
